### PR TITLE
Demo summary 8/2022

### PR DESCRIPTION
**Pull request by kpodemski**
Summary of the demo to be published tomorrow.

### CODE REVIEW

**Review by matks — APPROVED** [review]
Proof-read for me ✅ 

Note : I just published https://github.com/PrestaShop/prestashop.github.io/pull/1152 so maybe you want to add a publish time

I let you choose how to publish

File: news/_posts/public-oss-demo/2022-11-10-public-demo-8-2022.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+The project's public demo was held for the eighth time this year and [streamed live](https://www.youtube.com/watch?v=nRf0EQX3lbg) on November 9th, 2022.
+
+The session's agenda was as follows:
+
+- [00:00](https://www.youtube.com/watch?v=nRf0EQX3lbg) Beginning of the session: the introduction
+- [01:55](https://youtu.be/nRf0EQX3lbg?t=114) Project news: PrestaShop 8.0 is available!
+- [03:17](https://youtu.be/nRf0EQX3lbg?t=197) Demo: introducing new back office API by [@atomiix](https://github.com/atomiix)
+- [11:47](https://youtu.be/nRf0EQX3lbg?t=707) Demo: PrestaShop 8.1 and beyond by [@FabienVallon](https://github.com/FabienVallon)
+- [24:36](https://youtu.be/nRf0EQX3lbg?t=1476) Questions & Answers (do you have any questions? [Fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session).
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-11-10-public-demo-8-2022.md]:
```suggestion
- [24:36](https://youtu.be/nRf0EQX3lbg?t=1476) Questions & answers (do you have any questions? [Fill this form](https://forms.gle/FWazuZnXBtFPauFZ7) and we'll answer in the next session).
```

File: news/_posts/public-oss-demo/2022-11-10-public-demo-8-2022.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+You can use the timestamps to jump directly to the parts you want to watch.
+
+[Watch replay on YouTube](https://www.youtube.com/watch?v=nRf0EQX3lbg){: .btn .btn-lg .btn-info}
+{: .text-center}
+
+Also, you can take a look at the slides presented during the demo [here](https://docs.google.com/presentation/d/1tb-quO-_gwaelkersNz6Jb4X4EnakSNkqkhQQkE6wqE/edit?usp=sharing).
+
+We are available for feedback about the project and the software. We invite everyone to join us on [Slack](https://www.prestashop-project.org/slack/).
```
Comment (coffrep) [inline news/_posts/public-oss-demo/2022-11-10-public-demo-8-2022.md]:
```suggestion
We are available for feedback about the project and the software. We also invite everyone to join us on [Slack](https://www.prestashop-project.org/slack/).
```

**Review by coffrep — APPROVED** [review]
Just two suggestions and I'm approving, thanks team!

**Comment by kpodemski** [comment]
Thanks @matks @coffrep :)

**Pull request merged**